pkgs/development/python-modules/ormsgpack/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cargo,
+  rustPlatform,
+  rustc,
+}:
+
+buildPythonPackage rec {
+  pname = "ormsgpack";
+  version = "1.9.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aviramha";
+    repo = "ormsgpack";
+    rev = version;
+    hash = "sha256-lFKHXTYtYEjtu+nXemQnB0yjkFD69gr0n7XfJ1Hy3J0=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
SInce the [Cargo.lock is available upstream](https://github.com/aviramha/ormsgpack/blob/master/Cargo.lock), it would be nice if you could use `fetchCargoVendor`, to avoid adding the lock file to nixpkgs (see #327064)

Reply:
Done.